### PR TITLE
Microsoft Word CheatSheet

### PR DESCRIPTION
**Pull request by javathunderman**
Reviving #1514. 

Pull request form pulled from an old comment. 

What does your Instant Answer do?
It provides keyboard shortcuts for common tasks within Microsoft Word.

What is the data source for your Instant Answer? (Provide a link if possible)
http://www.dummies.com/how-to/content/word-2010-for-dummies-cheat-sheet.html

Why did you choose this data source?
The official and seemingly best source of information on keyboard shortcuts for Word. The layout is also helpful for users looking for more information as it's split into specific sections.

Are there any other alternative (better) data sources?
There are other resources, but I feel they are not better. The user will likely have more confidence with the official site also. Example alternative source is http://www.computerhope.com/shortcut/word.htm.

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
People who use Microsoft Word - Office & Home users.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
Yes it is - https://duck.co/ideas/idea/6318/microsoft-word-cheat-sheet

Are you having any problems? Do you need our help with anything?
No it's working :)

Where did you hear about DuckDuckHack? (For first time contributors)
Not 100% sure, I think it was a sponsored Facebook post.

IA Page: https://duck.co/ia/view/microsoft_word_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/microsoft-word.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            "key": "[Ctrl] [Shift] [>]"
+        }, {
+            "val": "Remove Paragraph or Character Formatting",
+            "key": "[Ctrl] [Spacebar]"
+        }, {
+            "val": "Paste Special",
+            "key": "[ALT] [Ctrl] [V]"
+        } 
+                         
+        ]}
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/microsoft-word.json]: Nit: Indentation is wrong here

Comment (javathunderman) [inline share/goodie/cheat_sheets/json/microsoft-word.json]: @mintsoft should be good!

File: share/goodie/cheat_sheets/json/microsoft-word.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
+{
+    "id": "microsoft_word_cheat_sheet",
+    "name": "Microsoft Word 2010",
+    "description": "Word Processor by Microsoft",
+    "metadata": {
+        "sourceName": "Dummies",
+        "sourceUrl": "http://www.dummies.com/how-to/content/word-2010-for-dummies-cheat-sheet.html"
+    },
+    "aliases": [
+        "ms word"
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/microsoft-word.json]: can you add `winword` here? Showing my age, but some people remember the old name!

Comment (javathunderman) [inline share/goodie/cheat_sheets/json/microsoft-word.json]: @mintsoft Haha, added that.

**Comment by mintsoft** [comment]
@javathunderman the cheatsheet source is reasonably large. Are you planning to incorperate more sections into this PR?

**Comment by javathunderman** [comment]
I don't think so...This cheatsheet covers most of the keyboard shortcuts. I don't see the need to get any more in.

**Comment by javathunderman** [comment]
@mintsoft Don't think I'll add more shortcuts seeing as that is what people use the most- I can add more if needed.

**Comment by javathunderman** [comment]
@moollaza Is there anything else that needs to be done for this goodie?

**Comment by javathunderman** [comment]
@mintsoft Updated with a new section to make things look less cumbersome.

**Comment by moollaza** [comment]
@javathunderman apologies for the delay, This LGTM :+1:

**Comment by javathunderman** [comment]
@moollaza cool, thanks!

**Comment by moollaza** [comment]
@javathunderman congrats! It's live: https://duckduckgo.com/?q=microsoft+word+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**